modular_splurt/code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -9,7 +9,7 @@
 		to_chat(usr, "<span class='warning'>You cannot toggle your gender while unconcious!</span>")
 		return
 
-	var/choice = tgui_alert(usr, "Select Gender.", "Gender", list("Both", "Male", "Female", "None"))
+	var/choice = tgui_alert(usr, "Select Gender.", "Gender", list("Both", "Male", "Female", "None","Toggle Breasts"))
```

Comment:
Not your fault but this code being duplicated is just bad.

Reply:
I'm not sure which part you are referring to- The fact that the code re-appears for simple mobs, robots, and PAI's, rather than just being one shared verb, or...?

Reply:
Yes, exactly. I haven't really looked into it but a shared verb would likely be the preferred way of doing this.